src/Components/CreateImageWizard/steps/Packages/PackageRecommendations.tsx

```diff
@@ -98,6 +99,7 @@ const PackageRecommendations = () => {
               isPreview: isBeta(),
               shownRecommendations: response.data.packages,
               selectedPackages: packages.map((pkg) => pkg.name),
+              distribution: distribution,
```

Comment:
I think we could maybe leave the distro in analytics as is, but it also might be nice to have the format consistent with the data we are sending :thinking:  Let's maybe update this and the analytics bellow to also remove the dash? Then it might be good to put the replace in a constant so we can just pass that.